akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/auction/AuctionEntity.java

```diff
@@ -27,7 +27,8 @@
   private final UUID entityUUID;
 
   public AuctionEntity(String entityId) {
-    // when used with Cluster Sharding this should use EntityTypeKey, or PersistentEntity
+    // when used with Cluster Sharding this should use EntityTypeKey, or
+    // PersistentEntity
```

Comment:
the other PR with the PersistenceId hasn't been merged yet, but this comment is anyway obsolete so great if you could remove it